File: Practice/python/LeapYear.py
# coding: utf-8

print("閏年かどうかを判定します：", end="")
year = int(input())

if (year%4 == 0 and year%100 != 0) or year%400 == 0:
    print("{}年は閏年です".format(year))
else:
    print("{}年は閏年ではありません".format(year))

File: Practice/ruby/LeapYear.rb
# coding: utf-8

print "閏年かどうかを判定します : "
year = gets.to_i

if (year%4 == 0 && year%100 != 0) || year%400 == 0
  puts "#{year}年は閏年です."
else
  puts "#{year}年は閏年ではありません."
end
